### PR TITLE
Awood/more mead

### PR DESCRIPTION
**Pull request by awood**
Adds several components missing from the Maven builds such as translations and the version.

### CODE REVIEW

File: gutterball/pom.xml [hunk 1]
```diff
@@ -821,6 +822,14 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.candlepin</groupId>
+      <artifactId>candlepin-common</artifactId>
+      <version>2.0.0</version>
+      <classifier>complete</classifier>
```
Comment (dgoodwin) [inline gutterball/pom.xml]: Does every common update require a manual version update in gutterball/candlepin?

File: pom.xml [hunk 2]
```diff
@@ -8,7 +8,8 @@
 
     <groupId>org.candlepin</groupId>
     <artifactId>candlepin-parent</artifactId>
-    <version>1.1</version>
+    <!-- This is the canonical version -->
+    <version>2.0</version>
```
Comment (dgoodwin) [inline pom.xml]: What is this version? Not following the canonical.

File: gutterball/pom.xml (hunk 1 above)
Comment (awood) [inline gutterball/pom.xml]: No, the `pom` plugin will take care of that.  After every update to common, you will need to run `buildr pom` but we should be doing that frequently anyway.

File: pom.xml (hunk 2 above)
Comment (awood) [inline pom.xml]: The version of "candlepin-parent".  It's not a version we use or a version that really matters much, but before the canonical location was in `buildfile`.  Now it's in `pom.xml`.

File: gutterball/pom.xml (hunk 1 above)
Comment (dgoodwin) [inline gutterball/pom.xml]: Works for me.

**Comment by dgoodwin** [comment]
Everything looks good to me Al, shall we merge?

**Pull request merged**